### DISCUSSION TITLE
May I ask when Ravyn OS can reach the level of Mac OS 10.15?

### DISCUSSION THREAD

**Discussion by qwt234** [ravynsoft/ravynos#420] [General]
It has been two years now. Could you please inquire about the current development of the system. How long until regular Mac users can use Ravyn OS that reaches Mac OS 10.15 or later on their PC?
Once again, thank you to the Ravyn OS team members for their hard work and efforts in breaking the system monopoly of MAC OS!

**Comment by mszoek** [comment]
Projects like this take a LONG time... consider Haiku and ReactOS, which are over 20 years old each and just recently getting fairly usable for running apps. I don't expect ravynOS to take 20 years because I didn't start from zero (that's why I chose to build on FreeBSD, Cocotron, and other existing projects) but it's still a long way from being ready for daily use.
